localtest/shared/uid2SdkTag.html

```diff
@@ -1 +1,2 @@
-<script src="https://cdn.integ.uidapi.com/uid2-sdk-3.5.0.js"></script>
+<!-- <script src="https://cdn.integ.uidapi.com/uid2-sdk-3.5.0.js"></script> -->
+<script src="../../dist/uid2-sdk-3.10.0.js"></script>
```

Comment:
where is this file coming from? i dont see the file in the repo

Reply:
I built it locally to test my changes.  I'll change it back.